### ISSUE TITLE
Invalid address used for login

### ISSUE THREAD

**Issue by DigitalDeviator** [dvandal/cryptonote-nodejs-pool#274]
Hi, I've got a bunch of miners on my pools who seem to find it very difficult to use the proper Wallet_Address.PaymentID to connect to the pools, generating an "Invalid address used for login" warning in the pools log files.

I'd think miners would check and fix this within a few minutes...However, in reality some miners simply don't...resulting in a continuous stream of failed login attempts every few seconds. 

It would be very helpful if the pool logs would also show a miner's IP Address with "Invalid address used for login" warnings. That way a pool admin can easily block them manually.

(or even better....if the pool would temporairly block them automatically after a set number of failed login attempts)  

Thanks for your time!

**Comment by avense** [comment]
it's wrong.
Walletadress + PaymentId and not Walletadress.PaymentId

**Comment by DigitalDeviator** [comment]
This is not wrong. In the Config.json you can manually set which "addressSeparator" you want to use for both PaymentID and fixedDiff....and this is very clearly stated on the "getting started" page.

Anyway...this has nothing to do with the issue of miners continuously making failed connecting attempts every few seconds. Which causes unnecessary load on the pool server. Also, this can easily be abused...Anyone can easily connect a whole bunch of "miners" with an invalid address to competing pools on purpose. Since the miner doesn't even need to actually mine it is even possible to make loads of those invalid connections from just one old computer.

**Comment by flychicken123** [comment]
Anyone know how does worker name work?I thought it was walletaddress+worker name

**Comment by DigitalDeviator** [comment]
workername you can set in the "password" field.

So instead of....  "-p x" or "--password x"
you can set "-p workername1"

**Comment by avense** [comment]
I think to make this change in config you also have to change in api.

So what do you want?
One way to stop failed logins?

**Comment by DigitalDeviator** [comment]
Clearly, you don't understand what I mean... or have no idea what you are talking about.

**Comment by avense** [comment]
@DigitalDeviator  I'm trying to help your stupid.

Fuck you

**Comment by bruceleeon** [comment]
@DigitalDeviator when you changed the + in the config to a . , did you change the difficulty from a . to something else?

**Comment by DigitalDeviator** [comment]
@bruceleeon Yes. the address separators are not the problem here. Please read my question again
